WordPress/Classes/ViewRelated/Blog/My Site/MySiteSettings.swift

```diff
@@ -13,19 +16,19 @@ import Foundation
     }
 
     @objc var experimentAssignment: String {
-        let defaultSection: MySiteViewController.Section = .siteMenu
-        let rawValue = userDefaults.object(forKey: Constants.defaultSectionKey) as? Int ?? defaultSection.rawValue
-        return MySiteViewController.Section(rawValue: rawValue)?.analyticsDescription ?? "nonexistent"
+        if let rawValue = userDefaults.object(forKey: Constants.defaultSectionKey) as? Int,
+            let defaultSection = MySiteViewController.Section(rawValue: rawValue)?.analyticsDescription {
+            return defaultSection
+        }
+
+        return "nonexistent"
     }
 
     // MARK: - Init
```

Comment:
[nit]
```suggestion
```

Reply:
Done in 5423c1458d295824d1888d9dc125eef4690a0aba